lib/FFI/CheckLib.pm

```diff
@@ -57,6 +57,36 @@ our $system_path = [];
 our $os ||= $^O;
 my $try_ld_on_text = 0;
 
+sub _homebrew_lib_path {
+  return () unless (qx`command -v brew`)[0];
+  chomp(my $brew_path = (qx`brew --prefix`)[0]);
+  return "$brew_path/lib";
+}
+
+sub _macports_lib_path {
+  my $port_path = (qx`command -v port`)[0];
+  return () unless $port_path;
+  chomp($port_path);
+  return $port_path =~ s|bin/port|lib|r;
+}
+
+sub _darwin_extra_paths {
+  my $pkg_managers = lc( $ENV{FFI_CHECKLIB_PACKAGE} // 'homebrew,macports' );
+  return () if $pkg_managers eq 'none';
+  my $supported_managers = {
+      homebrew => \&_homebrew_lib_path,
+      macports => \&_macports_lib_path
+  };
+  my @extra_paths = ();
+  foreach my $pkg_manager (split( ',', $pkg_managers )) {
```

Comment:
```suggestion
  foreach my $pkg_manager (split( /,/, $pkg_managers )) {
```

Perl::Critic policy violation here: https://github.com/PerlFFI/FFI-CheckLib/actions/runs/3091309566/jobs/5002332266#step:4:181

The TL;DR is that split actually treats this argument as a regex even if it comes in as a string, and I prefer it to explicitly look like a regex, even if as in this case there is no functional difference.

Reply:
Done!